.env.sample

```
@@ -1,6 +1,29 @@
+DOMAIN=http://localhost:3001
+REACT_APP_API_URL=http://localhost:4000
+REACT_APP_API_SUBSCRIPTION_URL=ws://localhost:4000/graphql
+REACT_APP_CDN_HOST=http://localhost:3001
+WIDGETS_DOMAIN=http://localhost:3200
+ALLOWED_ORIGINS=http://127.0.0.1:3001
+NGINX_HOST=localhost
+
+GATEWAY_PORT=4000
+UI_PORT=3001
+
+Docker:    MONGO_URL=mongodb://mongo:27017/erxes
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The Docker:    MONGO_URL key has incorrect delimiter

(IncorrectDelimiter)